### PR TITLE
Apply L1 phase calibration for PAR runs in correction framework

### PR DESCRIPTION
**Pull request by amatyja**
Apply L1 phase calibration for PAR runs in correction framework in AliEmcalCorrectionComponent.cxx.

### CODE REVIEW

File: PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.cxx [hunk 1]
```diff
@@ -208,9 +208,22 @@ void AliEmcalCorrectionComponent::UpdateCells()
...
+    Short_t currentParIndex = 0;
+    if(fRecoUtils->IsParRun()){
+      ULong64_t globalEventID = (ULong64_t)bunchCrossNo + (ULong64_t)fEventManager.InputEvent()->GetOrbitNumber() * (ULong64_t)3564 + (ULong64_t)fEventManager.InputEvent()->GetPeriodNumber() * (ULong64_t)59793994260;
+      for(Short_t ipar=0;ipar<fRecoUtils->GetNPars();ipar++){
+	if(globalEventID >= fRecoUtils->GetGlobalIDPar(ipar)) {
+	  currentParIndex++;
+	}
+      }
+    }
+    fRecoUtils->SetCurrentParNumber(currentParIndex);
```
Comment (loizides) [inline PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.cxx]: This was not here before so even if par is off why do we set it here?

Comment (amatyja) [reply PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.cxx]: currentParIndex is set to 0. And 0 means that the regular calibration before any PAR is taken.

Comment (amatyja) [reply PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.cxx]: In priniple it is safe to go with this setting into the condition statement.

Comment (loizides) [reply PWG/EMCAL/EMCALtasks/AliEmcalCorrectionComponent.cxx]: I would prefer the latter...

**Comment by loizides** [comment]
+1

**Pull request merged**